assets/sass/_theme/sections/sitemap.sass

```
@@ -10,4 +11,18 @@
                 @include media-breakpoint-up(desktop)
                     margin-bottom: 7.5rem
                 li
-                    margin-bottom: 1rem
\ No newline at end of file
+                    margin-bottom: 1rem
+.block-sitemap
+    ul 
+        list-style-type: none
+        li
+            position: relative
+            ul
+                margin-left: $spacing-3
+                li
+                    a
+                        @include icon(list-hyphen)
+                        &::before
```

@arnaudlevy tu peux enlever le sélecteur ```&::before```, car le mixin ```icon``` sert aussi de sélecteur

```suggestion
                        @include icon(list-hyphen)
                            position: absolute
                            left: pxToRem(-24)
                            font-size: pxToRem(24)
```